db/seeds.rb

```diff
@@ -147,92 +56,28 @@
-           Break and we stopped in for some fresh seafood and they delivered!
-           The oysters, Ipswich clams, arugula salad, North End Cioppino and
-           Lobster Spaghettini were all delicious.  Highly recommend the food
-           and the guys behind the counter/bar where we were seated provided
-           good service!",
-           "I came on a weekday so there was no wait! Ordered a glass of Chardonnay
-           and a dozen oysters wellfleets, katama bay, kumamamoto, and kusshi.
-           My favorites are kumamamotos and my new fav is the kusshi! So sweet
-           and all the oysters super fresh that I had to get 6 more.",
-           "We tried to eat here at around 8:30 on a Saturday night and were
-           told it was a two hour wait, and that we couldn't wait, because the
-           kitchen closes at 9:30. Closing is at 10:30. Soo...basically they told
-           us to get lost because they wanted to close the kitchen prior to what
-           they advertise as closing hours."]
-
 r_and_d = [true, false]
 
-zip_code = %w[86704
-48888
-12674
-79394
-19860
-64350
-39193
-43777
-52376
-68211
-44764
-64304
-36353
-41204
-31701
-26684
-44040
-01744
-69863
-82543]
+user = User.new(name: "admin", email: "sd@seeder.com", password: "seedpassword")
+user.save!
 
-description = ["They have some signature dishes like pumpkin tortellini
-            and butternut squash ravioli that must be tried to be believed.",
-           "My husband had the butternut squash ravioli and it was seriously
-           the most amazing taste!",
-           "I recommend the Cacio e Pepe pizza and the baked ricotta appetizer,
-           hands down.",
-           "We had the Cacio e pepe Pizza and a Margherita Pizza, they were delicious!",
-           "The dining area was very clean and has a view into the open kitchen.",
-           "I went with here with my wife for our anniversary
-           and she loved it but i thought it was the worst place ever",
-           "The yuzu kosho aioli rocked, and the squid ink was a powerful flavor,
-           a great addition to the oyster.",
-           "Our server was absolutely incredible and even gave us powdered-foie
-           gras on the house.",
-           "(Lucky 8) Foie Gras, balsamic chocolate kabayaki, raisin cocoa pulp,
-           sip of aged sake.",
-           "My favorite dish is the lobster roll -while a bit pricy,
-           it is the best and I always get it",
-           "I bring my chihuahua here and she likes the steak tartar
-           the most, but personally i think its disgusting"]
+client = Yelp::Client.new({ consumer_key: ENV['CONSUMER_KEY'],
+                            consumer_secret: ENV['CONSUMER_SECRET'],
+                            token: ENV['TOKEN'],
+                            token_secret: ENV['TOKEN_SECRET']
+                          })
 
-ratings = [1, 2, 3, 4, 5]
 
-50.times do
+result = client.search('chinatown, ma' , { term: 'chinese' })
```

Comment:
Redundant curly braces around a hash parameter.<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.